src/console_cmds.cpp

```diff
@@ -287,7 +302,7 @@ static bool ConResetTile([[maybe_unused]] uint8_t argc, [[maybe_unused]] char *a
 	}
 
 	if (argc == 2) {
-		auto result = ParseInteger(argv[1]);
+		auto result = ParseInteger(argv[1], 16);
```

Comment:
```suggestion
		auto result = ParseInteger(argv[1], 0);
```

`std::from_char` and thus `ParseInteger` behaves differently to C functions like `strtol`:
* `ParseInteger(..., 16)´ will only accept pure hex sequences without leading `0x`.
* `ParseInteger(.... 0)` will use `0x` to deceide between hex and decimal.

I think for manual input/human interfaces you basically never want the former, always the latter.

Reply:
For some reason I wasn't really thinking. After all, in the documentation for the `base` parameter I wrote down that `0` should be used for automatic detection. Yet, I didn't do that for these calls.